runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/kotlin/kotlinClasspathUtils.kt

```diff
@@ -55,8 +55,11 @@ private fun KotlinCompilation.classpathOf(project: Project): FileCollection {
 
     val shouldKeepBackwardsCompatibility = (kgpVersion != null && kgpVersion < KotlinGradlePluginVersion(1, 7, 0))
     return if (shouldKeepBackwardsCompatibility) {
-        @Suppress("DEPRECATION_ERROR")
-        kotlinCompile.classpath // deprecated with error since 1.8.0, left for compatibility with < Kotlin 1.7
+        // removed since 1.9.0, left for compatibility with < Kotlin 1.7
+        @Suppress("UNCHECKED_CAST")
+        val classpathGetter= kotlinCompile::class.members
+            .first { it.name == "getClasspath" }
+        classpathGetter.call(kotlinCompile) as FileCollection
```

Comment:
Looks like the suppress is for the wrong statement?
```suggestion
        val classpathGetter = kotlinCompile::class.members
            .first { it.name == "getClasspath" }
          
        @Suppress("UNCHECKED_CAST")
        classpathGetter.call(kotlinCompile) as FileCollection
```